core/src/main/scala/com/rackspace/com/papi/components/checker/util/VarXPathExpression.scala

```diff
@@ -0,0 +1,76 @@
+/***
+ *   Copyright 2016 Rackspace US, Inc.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+package com.rackspace.com.papi.components.checker.util
+
+import javax.xml.namespace.QName
+import javax.xml.xpath.XPathExpression
+import javax.xml.xpath.XPath
+import javax.xml.xpath.XPathVariableResolver
+
+import org.xml.sax.InputSource
+
+object VarXPathExpression {
+  def compile (xpath : XPath, expression : String,
+               vars : Map[QName, Object] = Map[QName, Object]()) : VarXPathExpression
+  = new VarXPathExpression(xpath, expression, vars)
```

Comment:
Just creating a factory to compile the expression as a convenience, but you're right not strictly required.